mat/cholesky.go

```diff
@@ -58,6 +71,48 @@ func (c *Cholesky) updateCond(norm float64) {
 	c.cond = 1 / v
 }
 
+// Dims returns the dimensions of the matrix.
+func (chol *Cholesky) Dims() (r, c int) {
+	if !chol.valid() {
+		panic(badCholesky)
+	}
+	r, c = chol.chol.Dims()
+	return r, c
+}
+
+// At returns the element at row i, column j.
+func (c *Cholesky) At(i, j int) float64 {
+	if !c.valid() {
+		panic(badCholesky)
+	}
+	n := c.Symmetric()
+	if uint(i) >= uint(n) {
+		panic(ErrRowAccess)
+	}
+	if uint(j) >= uint(n) {
+		panic(ErrColAccess)
+	}
+
+	var val float64
+	for k := 0; k <= min(i, j); k++ {
+		val += c.chol.at(k, i) * c.chol.at(k, j)
+	}
+	return val
+}
+
+// T implements the Matrix interface. Symmetric matrices, by definition, are
+// equal to their transpose, and this is a no-op.
+func (c *Cholesky) T() Matrix {
+	return c
+}
+
+// Symmetric implements the Symmetric interface and returns the number of rows
```

Comment:
```
// Symmetric implements the Symmetric interface and returns the number of rows
// and columns of the matrix.
```

Reply:
I originally had that, but thought it might be confused with "the sum of the rows and columns". If you aren't confused then I was overthinking it.